test/LibraryService.test.ts

```diff
@@ -0,0 +1,46 @@
+import { initialize, Server } from '@projecttacoma/node-fhir-server-core';
+import { serverConfig } from '../src/config/serverConfig';
+import { cleanUpDb, testSetup } from './utils';
+const supertest = require('supertest');
+
+let server: Server;
+
+const Library: fhir4.Library = {
+  resourceType: 'Library',
+  type: { coding: [{ code: 'logic-library' }] },
+  id: 'library123',
+  status: 'active'
+};
+
+describe('LibraryService', () => {
+  beforeAll(async () => {
+    const config = serverConfig;
+    server = initialize(config);
+    await testSetup([Library]);
+  });
+  describe('CRUD operations', () => {
+    test('test searchById with correctHeaders and the id should be in database returns 200', async () => {
+      await supertest(server.app)
+        .get('/4_0_1/Library/library123')
+        .set('Accept', 'application/json+fhir')
+        .set('content-type', 'application/json+fhir')
+        .expect(200)
+        .then((response: any) => {
+          expect(response.body.id).toEqual(Library.id);
+        });
+    });
+    test('test searchById when the id cannot be found in the database', async () => {
+      await supertest(server.app)
+        .get('/4_0_1/Library/invalidID')
+        .set('Accept', 'application/json+fhir')
+        .expect(404)
+        .then((response: any) => {
```

Comment:
When you install the supertest types, methinks this will no longer need the `any` type declaration